translations/translations.json

```
@@ -5686,418 +5686,277 @@
   },
   "intro": {
     "de": {
-      "appDescription": "{{appName}} ist Dein digitaler Guide für Deutschland",
-      "search": "Suche",
-      "searchDescription": "Anhand von Schlagwörtern schnell die richtige Information finden",
-      "languageChange": "Sprachwechsel",
-      "languageChangeDescription": "Einfaches Umstellen der Sprache einzelner Seiten",
+      "welcome": "Willkommen bei {{appName}}",
+      "appDescription": "In {{appName}} findest du Antworten und Tipps in deiner Muttersprache. Zum Beispiel: Wo finde ich Hilfe? Wann darf ich arbeiten?",
+      "search": "Einfache Suche",
+      "searchDescription": "Du findest schnell die richtigen Informationen über die Suche rechts oben im Menü.",
+      "languageChange": "Sprache verändern",
+      "languageChangeDescription": "Du kannst oben im Menü die Sprache von Integreat verändern.",
       "events": "Veranstaltungen",
       "eventsDescription": "Aktuelle Veranstaltungen und Termine in der Umgebung",
+      "directions": "Angebote in deiner Nähe",
+      "directionsDescription": "In der Karte kannst du Orte in deiner Nähe finden, zum Beispiel Orte für Sprachkurse.",
+      "offline": "Funktioniert auch offline",
+      "offlineDescription": "Du hast nicht immer Internet? Kein Problem. Du kannst Integreat auch ohne Internet benutzen.",
+      "information": "Aktuelle Informationen",
+      "informationDescription": "Was gibt es Neues in deiner Nähe?  Finde es heraus in Integreat.",
```

:wrench: There was a mean `LSEP` hidden in the translation:
```suggestion
      "informationDescription": "Was gibt es Neues in deiner Nähe? Finde es heraus in Integreat.",
```